data/json/snippets/missing_posters.json

```
@@ -5,57 +5,57 @@
     "text": [
       {
         "id": "male_1",
-        "text": "MISSING\nHAVE YOU SEEN THIS PERSON?\n\n[Photograph: professional headshot of a middle-aged man wearing glasses and a formal white shirt.]\n\nFULL NAME: Theodore Dean Garrity\nDOB: January 23, 1979\nAGE: 43\nSEX: Male\nHEIGHT: 6'1\"\nWEIGHT: 171 lbs\nRACE: Caucasian\nHAIR: Brown\nEYES: Hazel\nLAST SEEN: April 15th, around 5PM, near Derry gas station\nIDENTIFYING MARKS: heart tattoo on left forearm above wrists\nWEARING: light blue windbreaker, gray jeans, 'I <3 MASSACHUSETTS' cap, leather boots\n\nIF YOU HAVE ANY INFORMATION ABOUT THEODORE'S WHEREABOUTS, CONTACT LOCAL LAW ENFORCEMENT OR CALL (617)781-7979."
+        "text": "MISSING\nHAVE YOU SEEN THIS PERSON?\n\n[Photograph: professional headshot of a middle-aged man wearing glasses and a formal white shirt.]\n\nFULL NAME: Theodore Dean Garrity\nDOB: January 23, 1979\nAGE: 43\nSEX: Male\nHEIGHT: 6'1\"\nWEIGHT: 171 lbs\nRACE: Caucasian\nHAIR: Brown\nEYES: Hazel\nLAST SEEN: April 15th, around 5PM, near Derry gas station\nIDENTIFYING MARKS: heart tattoo on left forearm above wrists\nWEARING: light blue windbreaker, gray jeans, 'I <3 MASSACHUSETTS' cap, leather boots\n\nIF YOU HAVE ANY INFORMATION ABOUT THEODORE'S WHEREABOUTS, CONTACT LOCAL LAW ENFORCEMENT OR CALL (617)555-7979."
```

```suggestion
        "text": {
            "str": "MISSING\nHAVE YOU SEEN THIS PERSON?\n\n[Photograph: professional headshot of a middle-aged man wearing glasses and a formal white shirt.]\n\nFULL NAME: Theodore Dean Garrity\nDOB: January 23, 1979\nAGE: 43\nSEX: Male\nHEIGHT: 6'1\"\nWEIGHT: 171 lbs\nRACE: Caucasian\nHAIR: Brown\nEYES: Hazel\nLAST SEEN: April 15th, around 5PM, near Derry gas station\nIDENTIFYING MARKS: heart tattoo on left forearm above wrists\nWEARING: light blue windbreaker, gray jeans, 'I <3 MASSACHUSETTS' cap, leather boots\n\nIF YOU HAVE ANY INFORMATION ABOUT THEODORE'S WHEREABOUTS, CONTACT LOCAL LAW ENFORCEMENT OR CALL (617)555-7979.",
            "//~": "Please replace the fictitious phone number with one of your local equivalent when translating"
        }
```
An example of how to add a translator comment. It will be extracted by the string extraction script and shown in Transifex along with the untranslated string, so translators will know how to translate it (if they pay attention to the comment, that is).